webhooks/virtualmachine/v1alpha1/validation/virtualmachine_validator.go

```diff
@@ -130,7 +130,6 @@ func (v validator) ValidateCreate(ctx *context.WebhookRequestContext) admission.
 	fieldErrs = append(fieldErrs, v.validateInstanceStorageVolumes(ctx, vm, nil)...)
 	fieldErrs = append(fieldErrs, v.validatePowerStateOnCreate(ctx, vm)...)
 	fieldErrs = append(fieldErrs, v.validateNextRestartTimeOnCreate(ctx, vm)...)
-	fieldErrs = append(fieldErrs, v.validateAnnotation(ctx, vm, nil)...)
```

Comment:
Isn't the goal to only drop the validation of the annotations set in the mutator?